scripts/bash/run_enceladus.sh

```diff
@@ -386,33 +401,37 @@ SPARK_CONF="--conf spark.logConf=true"
 
 # Dynamic Resource Allocation
 # check DRA safe prerequisites
-if [ "$DRA_ENABLED" = true ] ; then
-    if [ -n "$NUM_EXECUTORS" ]; then
-        echo "WARNING: num-executors should NOT be set when using Dynamic Resource Allocation. DRA is disabled.";
-        DRA_ENABLED=false
-    fi
-    if [ -z "$DRA_MAX_EXECUTORS" ]; then
-        echo "WARNING: maxExecutors should be set for Dynamic Resource Allocation. DRA is disabled"
-        DRA_ENABLED=false
-    fi
+if [ "$DRA_ENABLED" = true ] && [ -z "$DRA_MAX_EXECUTORS" ] ; then
+    echo "WARNING: maxExecutors should be set for Dynamic Resource Allocation. DRA is disabled"
+    DRA_ENABLED=false
 fi
 
 # configure DRA and adaptive execution if enabled
 if [ "$DRA_ENABLED" = true ] ; then
-    echo "Dynamic Resource Allocation enabled"
-    add_spark_conf_cmd "spark.dynamicAllocation.enabled" "true"
-    add_spark_conf_cmd "spark.shuffle.service.enabled" "true"
-    add_spark_conf_cmd "spark.sql.adaptive.enabled" "true"
-    add_spark_conf_cmd "spark.dynamicAllocation.maxExecutors" "${DRA_MAX_EXECUTORS}"
-    if [ -n "$DRA_MIN_EXECUTORS" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.minExecutors" "${DRA_MIN_EXECUTORS}"
-    fi
-    if [ -n "$DRA_ALLOCATION_RATIO" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.executorAllocationRatio" "${DRA_ALLOCATION_RATIO}"
-    fi
-    if [ -n "$ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE" ]; then
-        add_spark_conf_cmd "spark.sql.adaptive.shuffle.targetPostShuffleInputSize" "${ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE}"
-    fi
+  echo "Dynamic Resource Allocation enabled"
+
+  if [ -n "$NUM_EXECUTORS" ]; then
+      echo "WARNING: num-executors should NOT be set when using Dynamic Resource Allocation and will be ignored."
```

Comment:
I think nobody will notice this warning. Don't we want to stop the script maybe?

Reply:
Discussed and clarified.